site/_posts/2017-10-16-how-was-your-last-week.md

```diff
@@ -0,0 +1,166 @@
+
+**Is it beneficial?**
+
+Yes, definitely. Without events, you can miss situation that container has been started and stopped (in short time) 
+between 2 refresh and usage of events could handle this situation.
+
+**When you remove container, is it removed from database?**
+
+No. We disconnect it and it means that container is archived.
+
+Thanks Ladas! 
+
+---
+
+**Hi [Zita][znemecko],
+how was your last week in ManageIQ ?**
+
+I was working on custom buttons as I added before and I was finishing some last things like a renaming and fixing.
+Also I was adding new ones related to users, tenants and container nodes.
+
+**Nice. What else? I heard about that your open source contribution is not only about the code.**
+
+Yes. Me and [MartinH][himdel] are teaching students on some university in Brno. Last week we had a lesson where we teach them about git and
+creaing pull requests.
+
+**How it is related to open source?**
+
+We are showing them our open source project and we are apprising them with for example with code reviews. We are trying 
+to show them more practical view on open source development. Most likely some of them will became contributors of ManageIQ :)
+
+Very good, thanks Zita! 
+
+---
+
+**Hi [MartinM][Hyperkid123], tell me something about yourself first.**
+
+I am working in UI team and I am intern.
+
+**How was your last week in ManageIQ ?**
+
+I was working topology graphs.
+
+**What did you do with them?**
+
+I was converting them from SVG to CANVAS. Also I am encapsulating them to standalone angular component.
+
+**Is it good ?**
+
+Yes, certainly. You can have one angular component instead of some angular controller. You can also test it nicely.
+
+**Will it change also visual page ?**
+
+Yes, look on it:
+
+![New Topology](/assets/images/blog/new-topology.png)
+
+Thanks Martin, good news! 
+
+---
+
+**Hi [David][skateman],
+how was your last week ?**
+
+I did some fixes for our trees. After the conversion to Angular some regressions appeared.
+
+**It is nice that we are using angular components more and more.**
+
+Yes I was also creating tests for angular components as well. Y'know, tests in javascript.
+
+**Do you like it?**
+
+No.
+
+**:D**
+
+**What else ?**
+
+I was preparing a demo for [Open Source Summit Europe][summit].
+
+**What will be demo about?**
+
+We will show ManageIQ to the attendees in a containerized environment.
+
+**How ?**
+
+We have an oVirt cluster on a set of Intel NUC boxes up and running now I'm deploying some VMs to run OpenShift. 
+ManageIQ will be running as a container.
+
+![NUC with ManageIQ](/assets/images/blog/nuc.jpg)
+
+Very good. Thanks David!
+
+---
+
+**Hi [MartinP][martinpovolny],
+how was your last week ?**
+
+I was mostly doing PR reviews and merging PRs.
+
+**Can you highlight on some of them ?**
+
+Yes. [Aparna][Aparna] did lot of work on generic objects UI.
+
+[Aparna][Milan] was simplifying i18 strings. It will help the translators.
```

Comment:
I think this is the last thing :-)  I think you meant to use Milan and not Aparna.

Reply:
:D yes, fixed